setup.py

```diff
@@ -67,7 +67,7 @@
             "lgpatch = legate.lgpatch:main",
         ],
     },
-    scripts=["bind.sh"],
+    scripts=["bind.sh", "gen_library"],
```

Comment:
This puts a `gen_library` binary in the user's PATH. Can we please call it something more descriptive? It'd be nice to rename `bind.sh`, but that ship may have sailed.

Reply:
Something like `python -m create-legate-library <flags>` would be ideal (cc: @bryevdv for any other ideas).

Reply:
`bind.sh` is internal (used in the invocation that the legate driver constructs), so we can rename it if it makes sense.

Reply:
maybe `init-legate-library`? (or `init-legate-lib` if more brevity is desired) But "create" would also be ok. 

Renaming generic `bind.sh` also seems reasonable, but let's leave that to a dedicated PR since there will be driver tests to update.

Reply:
done, renamed to `create-legate-library`